api/src/main/resources/workbench.yaml

```diff
@@ -922,6 +922,249 @@ paths:
           schema:
             $ref: "#/definitions/CohortSummaryListResponse"
 
+  /api/v1/workspaces/{workspaceNamespace}/{workspaceId}/review/{cohortId}/{cdrVersionId}/annotationdefinitions:
+    post:
+      tags:
+        - cohortReview
```

Comment:
You could add a new controller / tag for annotations. (I could see  going either way.)

Reply:
See comment below about participantCohortAnnotation.